pkg/subctl/cmd/gather/logs.go

```diff
@@ -31,6 +31,9 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+var PodLogInfos []logInfo
+var podLogInfo logInfo
```

Comment:
Rather than using globals to communicate results for the summary, I think it would be cleaner to create a `Summary` struct and pass that to the various functions to populate via the `Info` struct.

Reply:
To think of it, the variables need not be global at all.